srcpkgs/a4/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'a4'
+pkgname=a4
+version=0.2.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="libtool pkg-config"
+makedepends="libvterm-devel libtickit-devel"
+depends="libvterm libtickit"
```

Comment:
Those `depends` are automatically detected through the shared library check and don't need to be specified.